public/app/features/panel/metrics_panel_ctrl.ts

```diff
@@ -166,6 +166,23 @@ class MetricsPanelCtrl extends PanelCtrl {
       return this.$q.when([]);
     }
 
+    var targets = JSON.stringify(this.panel.targets);
+    var intervalVar = null;
+    _.each(this.templateSrv.variables, function(variable) {
```

Comment:
Align time range cause extra data points fetch which isn't plotted on screen.
So, avoid such fetch for the panels which doesn't use "interval" variable.